docs/initialize-app.md

```diff
@@ -16,7 +16,9 @@ amplify init
 
 2. Press Enter to accept the default project name ‘notesapp’
 
-3. Prese enter for the default environment name 'dev'
+3. Select 'n' when asked to 'Initialize the project with the above configuration' 
```

Comment:
Looks like the UI flow for this has changed slightly - you need to say 'no' to this new prompt to get to the bit where you can fill the rest you need in